docs/docs.less

```diff
@@ -171,14 +171,5 @@ min-version {
     }
```

Comment:
Please merge and rebase `master` to resolve file conflicts:

<img width="748" alt="Screen Shot 2020-11-17 at 1 41 14 PM" src="https://user-images.githubusercontent.com/10120600/99439498-debf6980-28da-11eb-9261-cf92ec41258e.png">

Reply:
Changes have been done. Please review